hyperspy/_components/polynomial.py

```diff
@@ -146,13 +145,11 @@ def convert_to_polynomial(poly_dict):
     Convert the dictionary from the old to the new polynomial definition
```

Comment:
Similar as below: log this "conversion" in the logger at the INFO level to avoid doing things silently?

Reply:
is already logged by the `convert_to_polynomial` function